### PR TITLE
feat(ramp): offramp welcome screen copy updates

### PR DESCRIPTION
**Pull request by georgeweiler**
## **Description**

The welcome page for sell has different copy requirements. This PR updates copy based on `rampType`

## **Related issues**
N/A

## **Manual testing steps**
Check the sell version of the welcome page and verify that the copy has been updated

### **Before**
Screenshot 2023-11-08 at 11 19 45 AM

### **After**
Screenshot 2023-11-08 at 11 18 47 AM

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've clearly explained what problem this PR is solving and how it is solved.
- [x] I've linked related issues
- [x] I've included manual testing steps
- [x] I've included screenshots/recordings if applicable
- [x] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [x] I’ve properly set the pull request status:
  - [x] In case it's not yet "ready for review", I've set it to "draft".
  - [x] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by wachunei** [comment]
I will convert this into draft until tests are done

**Review by wachunei — APPROVED** [review]
LGTM

File: locales/languages/en.json [hunk 1]
```diff
@@ -1906,7 +1906,9 @@
     "onboarding": {
       "what_to_expect": "What to Expect",
       "quotes": "Our buy crypto feature aggregates quotes from integrated vendors, providing quotes from those sources to get crypto directly into your wallet with no waiting period.",
+      "quotes_sell": "Now you can cash out directly in MetaMask! Get up-to-the-minute quotes from trusted providers while we guide you every step of the way.",
```
Comment (Cal-L) [inline locales/languages/en.json:1909]:
```suggestion
      "quotes_sell": "You can now cash out directly in MetaMask! Get up-to-the-minute quotes from trusted providers while we guide you through every step of the way.",
```
Nit to flow a bit better

**Review by Cal-L — COMMENTED** [review]
left one comment

File: locales/languages/en.json (hunk 1 above)
Comment (Cal-L) [reply locales/languages/en.json:1909]: But feel free to address only if needed

**Review by Cal-L — APPROVED** [review]
LGTM

**Comment by bkirb** [comment]
@georgeweiler LGTM I verified this change ✅

**Pull request merged**